tests/tests/test_backward_compatibility/test_import.py

```diff
@@ -101,7 +97,6 @@ def test_21(verbose=False, plot=False):
     if plot:
         b.plot(show=True, time=0)
 
-    return b
```

Comment:
are these intentionally removed (in several places throughout this PR)?  They can be useful when running the script directly and debugging the bundle.  If we do need to remove these, then the calls in `__main__` need to be updated as well.

Reply:
(turns out this is necessary otherwise pytest yells, so we'll just need to manually make these edits if we need to inspect the bundle to debug a broken test).

Reply:
Yep, pytest currently raises a warning but the next version will raise an error whenever tests return anything. So we can't put those back in, unfortunately. Here [here](https://github.com/pytest-dev/pytest/issues/7337) for details.